src/main/kotlin/com/terning/server/kotlin/ui/api/ScrapController.kt

```diff
@@ -0,0 +1,38 @@
+package com.terning.server.kotlin.ui.api
+
+import com.terning.server.kotlin.application.ScrapService
+import com.terning.server.kotlin.application.scrap.ScrapRequest
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/v1/scraps")
+class ScrapController(
+    private val scrapService: ScrapService,
+) {
+    @PostMapping("/{internshipAnnouncementId}")
+    fun scrap(
+        // @AuthenticationPrincipal userId: Long,
+        @PathVariable internshipAnnouncementId: Long,
+        @RequestBody scrapRequest: ScrapRequest,
+    ): ResponseEntity<ApiResponse<Unit>> {
+        val userId: Long = 1 // 임시 userId
```

Comment:
주석은 TODO로 적어두면 좋을 것 같아요!
```suggestion
        val userId: Long = 1 // TODO : 임시 userId
```

Reply:
좋네요! (할까 말까 고민했었는데,, 덕분에 잘 챙겨갑니다~ 😊)
TODO로 명시해두면 나중에 수정해야 할 부분을 더 분명하게 파악할 수 있어서 확실히 도움이 될 것 같아요.
말씀 주신 대로 // TODO: 임시 userId 형태로 반영하겠습니다! 감사합니다 🙏✨